usaspending_api/search/v2/views/spending_over_time.py

```diff
@@ -208,22 +256,58 @@ def parse_elasticsearch_bucket(self, bucket: dict) -> dict:
             "insurance": "Other_Obligations",
         }
 
+        # Outlays are only supported on Awards
+        if self.spending_level == "awards":
+            outlay_dictionary = {
+                "Contract_Outlays": 0,
+                "Direct_Outlays": 0,
+                "Grant_Outlays": 0,
+                "Idv_Outlays": 0,
+                "Loan_Outlays": 0,
+                "Other_Outlays": 0,
+            }
+        else:
+            outlay_dictionary = {
+                "Contract_Outlays": None,
+                "Direct_Outlays": None,
+                "Grant_Outlays": None,
+                "Idv_Outlays": None,
+                "Loan_Outlays": None,
+                "Other_Outlays": None,
+            }
```

Comment:
You could use the values in the `outlay_map` which has the advantage of not having to repeat these values and worry about keeping them in sync. 

```python 
outlay_dictionary = {v: 0 if self.spending_level == "awards" else None for v in outlay_map.values}
```

Reply:
Good suggestion, I forgot these field names were available in the `outlay_map` dictionary. Updated the code.